### DISCUSSION TITLE
Is it possible to only use the themes section of daisyui?

### DISCUSSION THREAD

**Discussion by scottromack** [saadeghi/daisyui#1400] [Q&A] [answered]
I just want to add a darkmode to a standard tailwind 3 project?

Thanks,

**Comment by saadeghi** [comment, marked as answer]
If you're using Tailwind CSS class names (like `gray-200` etc) these class names apply a hardcoded color value. `gray-200` would always be light gray and `gray-800` would always be dark gray.  
So if your div is `<div class="bg-gray-200">` you can't make it dark unless you add additional class names for dark mode: `<div class="bg-gray-200 dark:bg-gray-800">` ([read more](https://tailwindcss.com/docs/dark-mode))  

Without daisyUI you need to add an additional `dark:` class name for all the color names you already have.  
daisyUI colors make it easier because daisyUI color are actually CSS variables so with daisyUI when we use `<div class="bg-base-100">` it can be light background on light theme and it can be dark background on dark theme, without `dark:` class or any other additional class names for color.

**If you want daisyUI themes on Tailwind CSS project:**  
You have to replace all color class names (all colors that need to change based on the theme) with daisyUI color names.  

Read this page to learn more about colors: https://daisyui.com/docs/colors/  
Also read this discussion: https://github.com/saadeghi/daisyui/discussions/640

**Comment by scottromack** [comment]
Thanks,
I do prefer your semantic color naming. It makes themes much easier.
